portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a favorite quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFavoriteQuote() {
+  const quotes =
```

Comment:
It's good to keep things consistent across the code and comments so I'd suggest renaming all of these to "favoriteQuotes" and "favoriteQuote" everywhere, or just have them as "quotes" and "quote" in all places. But the idea is to stick to one.